### PR TITLE
Add TypeScript section to troubleshooting guide

### PR DESCRIPTION
**Pull request by daogrady**
This PR adds a section in the troubleshooting guide for TypeScript, starting with one frequent complaint that arose since the release of CDS8.

### CODE REVIEW

**Comment by renejeglinsky** [comment]
Hi @bugwelle ,
I tried to [fix some snippets](https://github.com/cap-js/docs/pull/1438/commits/405a026753f7336c067a3928cdd75352e18d7983) but only was partially successful. Can you check how those snippets need to be adjusted?

File: get-started/troubleshooting.md [hunk 1]
```diff
@@ -191,6 +191,36 @@ module.exports = cds.server
...
+::: code-group
+```sh [facet]
+cds add typescript
+```
+```sh [manually]
+npm i -D @cap-js/cds-types
+```
+:::
+
+Installing `@cap-js/cds-types` leverages vscode's automatic type resolution mechanism by symlinking the package in `node_modules/@types/sap__cds` in a postinstall script. If you find that this symlink is missing, try `npm rebuild` to trigger the postinstall script again.
```
Comment (renejeglinsky) [inline get-started/troubleshooting.md:209]:
```suggestion
Installing `@cap-js/cds-types` leverages VS Code's automatic type resolution mechanism by symlinking the package in `node_modules/@types/sap__cds` in a postinstall script. If you find that this symlink is missing, try `npm rebuild` to trigger the postinstall script again.
```

File: get-started/troubleshooting.md [hunk 2]
```diff
@@ -191,6 +191,36 @@ module.exports = cds.server
...
+
+::: code-group
+```sh [facet]
+cds add typescript
+```
+```sh [manually]
+npm i -D @cap-js/cds-types
+```
+:::
+
```
Comment (renejeglinsky) [inline get-started/troubleshooting.md:208]:
```suggestion

#### Fix missing symlink

```

Comment (daogrady) [reply get-started/troubleshooting.md:217]: is the imperative mood correct here/ a guideline? Maybe this is just personal taste, but "Fixing missing symlink" feels more natural.

Comment (daogrady) [reply get-started/troubleshooting.md:217]: nvm, was already resolved, my bad.

**Review by chgeo — APPROVED** [review]

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**